db/migrate/20210331094618_create_reviews.rb

```diff
@@ -0,0 +1,13 @@
+class CreateReviews < ActiveRecord::Migration[5.2]
+  def change
+    create_table :reviews do |t|
+      t.integer :listing_id
```

Comment:
No need to add `listing_id` to the Reviews table.
This can be easily accessed through the `booking`.
Please use model associations wisely.